modules/tracing/mule-tracer-exporter-impl/src/main/java/org/mule/runtime/tracer/impl/exporter/OpenTelemetrySpanExporter.java

```diff
@@ -104,42 +232,186 @@ public void updateChildSpanExporter(SpanExporter childSpanExporter) {
       }
 
       // No export if no export till reset.
```

Comment:
```suggestion
      // Do not export if noExportUntil is set.
```
is this what you mean?

Reply:
I have added this:

// In case "no export until" is set, and it is not a child span that resets that condition (because
      // we have a span that begins again to be exportable), we have to propagate that condition to the
      // child span.